src/Index.ts

```diff
@@ -152,11 +167,11 @@ async function messageHandler(msg: Message) {
 		}
 	} catch (error) {
 		console.log({ error });
-		msg.channel.send(Embed.error(error));
+		msg.channel.send({ embeds: [Embed.error(error)] });
```

Comment:
is this a promise that should be awaited? i forgor

Reply:
It looks like it actually is a promise, although it seems like we are not awaiting this like anywhere lol

Reply:
It does return a promise but is there any point in awaiting it if we're not using/depending on the return value anywhere?